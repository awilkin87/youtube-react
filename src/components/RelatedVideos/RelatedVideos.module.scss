.relatedVideos {
  & > * {
    margin-top: 4px;
  }
}
