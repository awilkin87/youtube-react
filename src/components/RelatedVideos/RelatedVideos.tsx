import VideoPreview from '../VideoPreview/VideoPreview';
import NextUpVideo from './NextUpVideo/NextUpVideo';
import styles from './RelatedVideos.module.scss';

export default function RelatedVideos() {
  return (
    <div className={styles.relatedVideos}>
      <NextUpVideo />
      <VideoPreview orientation='horizontal' />
      <VideoPreview orientation='horizontal' />
      <VideoPreview orientation='horizontal' />
    </div>
  )
}
