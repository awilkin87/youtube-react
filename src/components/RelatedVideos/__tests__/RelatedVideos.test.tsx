import { shallow } from "enzyme";
import RelatedVideos from '../RelatedVideos';

test('Renders RelatedVideos', () => {
  const wrapper = shallow(<RelatedVideos />);
  expect(wrapper).toMatchSnapshot();
});
