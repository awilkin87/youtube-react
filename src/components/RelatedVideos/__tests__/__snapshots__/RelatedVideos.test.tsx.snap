// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders RelatedVideos 1`] = `
<div
  className="relatedVideos"
>
  <NextUpVideo />
  <VideoPreview
    orientation="horizontal"
  />
  <VideoPreview
    orientation="horizontal"
  />
  <VideoPreview
    orientation="horizontal"
  />
</div>
`;
